src/builder/pod/mod.rs

```diff
@@ -163,6 +163,16 @@ impl PodBuilder {
         self
     }
 
+    /// Utility function for the common case of adding an emptyDir Volume
+    /// with the given name and no medium and no quantity.
+    pub fn add_empty_dir_volume(&mut self, name: impl Into<String>) -> &mut Self {
+        self.add_volume(
+            VolumeBuilder::new(name)
+                .with_empty_dir(None::<String>, None)
+                .build(),
+        )
+    }
```

Comment:
We should at least be able to set a size? Meaning pass through the `quantity: Option<Quantity>`

Reply:
I think the majority of cases doesn't use a size, and I'd like to keep it tidy. why not have two functions in that case? We can have as many as we want!

Reply:
I dont see the point for that. `with_empty_dir` already takes a quantity, why not pass it through instead of setting it to None  and/or create another method?

Reply:
Otherwise if I'm reading the code I gotta think about this `None` parameter. If there is nothing its easier to read/think about

Reply:
I mean you have the prototype of that function, you can but do not have to specify a quantity (which will default to whatever). I dont think this is hard to read :D